lms/static/sass/course/_info.scss

```diff
@@ -31,8 +81,9 @@ div.info-wrapper {
         }
 
         h2 {
-          font-size: $body-font-size;
-          font-weight: bold;
+          @extend %t-title9;
+          margin-bottom: ($baseline/4);
+          text-transform: none;
           background: url('#{$static-path}/images/calendar-icon.png') 0 center no-repeat;
           padding-left: $baseline;
```

Comment:
I know this isn't part of your commit, but should this also be `@include padding-left`